micrometer-observation/src/test/java/io/micrometer/observation/aop/ObservedAspectTests.java

```diff
@@ -60,7 +59,7 @@ void annotatedCallShouldBeObserved() {
                 .hasNameEqualTo("test.call").hasContextualNameEqualTo("test#call")
                 .hasLowCardinalityKeyValue("abc", "123").hasLowCardinalityKeyValue("test", "42")
                 .hasLowCardinalityKeyValue("class", ObservedService.class.getName())
-                .hasLowCardinalityKeyValue("method", "call").thenThrowable().doesNotThrowAnyException();
+                .hasLowCardinalityKeyValue("method", "call").thenError().doesNotThrowAnyException();
```

Comment:
in this class, there's a bunch of `.thenError().doesNotThrowAnyException()` (which reads weird). could now be converted to `.doesNotHaveError()`, wdyt?